src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -109,7 +109,7 @@ public function __construct(array $normalizers = [], array $encoders = [])
     /**
      * {@inheritdoc}
      */
-    final public function serialize($data, $format, array $context = [])
+    public function serialize($data, string $format, array $context = [])
```

Comment:
no pb on my side, but why removing `final`?

Reply:
That was a C&P fail. Reverted.